module/surveys/helper.js

```diff
@@ -1808,4 +1809,401 @@ module.exports = class SurveysHelper {
         })
     }
 
+    static getDetailsByLink2(link= "", userId= "", token= "", roleInformation= {},version = "") {
```

Comment:
@swetha-aggidevara are we not using existing functions?

Reply:
Existing function is being used by other api's and this function gets the data from API